src/utils/openwhisk.js

```diff
@@ -167,7 +168,9 @@ async function runPipeline(cont, pipe, actionParams) {
   if (actionParams.__ow_logger && !actionParams.__ow_logger.trace) {
     actionParams.__ow_logger.trace = actionParams.__ow_logger.silly;
   }
-  return logger(logger.trace(runner))(actionParams);
+
+  const epsagonWrap = epsagon(runner, actionParams);
+  return logger(logger.trace(epsagonWrap))(actionParams);
```

Comment:
you could also use `openwhisk-action-utils.wrap` for simplicity.

Reply:
I hesitated but I though saving a dependency will be smarter (this is all packaged during `hlx build`).

Reply:
If it doesn't increase the bundle size of an example action by more than 1%, I'd go with @tripodsan's proposal.

Reply:
zipped or not zipped ? :) It adds 3.15% (zipped)!!
But since we are only talking about 100Ko zipped and I like clean code, I will still do it.

Reply:
> zipped or not zipped ? :) It adds 3.15% (zipped)!!

just the `wrap` or adding `epsagon` ?

Reply:
`wrap` and its dependencies.

Reply:
> `wrap` and its dependencies.

hmm...weird.... I have to investigate this....